flo-runner/src/main/java/com/spotify/flo/context/ForkingContext.java

```diff
@@ -26,55 +26,23 @@
 import com.spotify.flo.Tracing;
 import io.grpc.Context;
 import java.io.IOException;
-import java.lang.management.ManagementFactory;
 import java.util.Collections;
-import java.util.Optional;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * An {@link EvalContext} that runs tasks in sub-processes.
  * <p>
- * Forking can be disabled using the environment variable {@code FLO_DISABLE_FORKING=true}.
- * <p>
- * Forking is disabled by default when running in the debugger, but can be enabled by {@code FLO_FORCE_FORK=true}.
- * <p>
  * The basic idea here is to intercept the process fn by overriding {@link EvalContext#invokeProcessFn(TaskId, Fn)},
  * and executing it in a sub-process JVM. The process fn closure and the result is transported in to and out of the
  * sub-process using serialization.
  */
 class ForkingContext extends ForwardingEvalContext {
 
-  private static final Logger log = LoggerFactory.getLogger(ForkingContext.class);
-
   private ForkingContext(EvalContext delegate) {
     super(delegate);
   }
 
   static EvalContext composeWith(EvalContext baseContext) {
-    // Is the Java Debug Wire Protocol activated?
-    final boolean inDebugger = ManagementFactory.getRuntimeMXBean().
-        getInputArguments().stream().anyMatch(s -> s.contains("-agentlib:jdwp"));
-
-    final Optional<Boolean> forking = Optional.ofNullable(System.getenv("FLO_FORKING")).map(Boolean::parseBoolean);
-
-    if (forking.isPresent()) {
-      if (forking.get()) {
-        log.debug("Forking enabled (environment variable FLO_FORKING=true)");
-        return new ForkingContext(baseContext);
-      } else {
-        log.debug("Forking disabled (environment variable FLO_FORKING=true)");
-        return baseContext;
-      }
-    } else if (inDebugger) {
-      log.debug("Debugger detected, forking disabled by default "
```

Comment:
I think we'd want to preserve the behavior or disabling forking by default if running in the debugger.